### DISCUSSION TITLE
Unable to connect to tkey libraries

### DISCUSSION THREAD

**Discussion by DanRizwa** [Web3Auth/Web3Auth#485] [Help]
I am trying to connect tkey module into a crypto project I am working on. The issue I am facing when connecting to tkey libraries is getting TorusServiceProvider error. Tried a lot of things but seems that I am stuck here

**Comment by yashovardhan** [comment]
Hey @DanRizwa 

Thanks for your question and sorry for the delay from our side in getting back to you. 

Did you check our documentation on Torus Service Provider? We have mentioned things around how to configure your Service Provider for usage with TKey in the Web3Auth Self Host SDK Reference: https://web3auth.io/docs/sdk/self-host/service-provider

Additionally, please have a look at our [self host guide](https://web3auth.io/docs/guides/selfhost) and [sample app](https://github.com/tkey/tkey-example)

If you still face problem, please share with us your code snippets and errors.
